python/lsst/pex/config/listField.py

```diff
@@ -506,7 +507,7 @@ def _compare(self, instance1, instance2, shortcut, rtol, atol, output):
         if not compareScalars(f"size for {name}", len(l1), len(l2), output=output):
             return False
         equal = True
-        for n, v1, v2 in zip(range(len(l1)), l1, l2):
+        for n, v1, v2 in zip(range(len(l1)), l1, l2, strict=False):
```

Comment:
Isn't `strict=False` the default?  And again, we know that they're the same length due to the `compareScalars` above.

Reply:
Yes, I can change it to strict=True. Sorry I merged after your comments came.